docs/platforms/javascript/common/user-feedback/configuration/index.mdx

```diff
@@ -77,70 +85,107 @@ feedbackIntegration({
+
+| `inputForeground`         | `--input-foreground`           | `inherit`                                 | `inherit`                                 | Foreground color for form inputs                                                                      |
+| `inputBackground`         | `--input-background`           | `inherit`                                 | `inherit`                                 | Background color for form inputs                                                                      |
+| `inputBackgroundFocus`    | `--input-background-focus`     | `inherit`                                 | `inherit`                                 | Background color for form inputs when focused                                                         |
+| `inputBorder`             | `--input-border`               | `var(--border)`                           | `var(--border)`                           | Border styles for form inputs                                                                         |
+| `inputBorderRadius`       | `--input-border-radius`        | `6px`                                     | `6px`                                     | Border radius style for form content (for example: inputs, buttons)                                   |
+| `inputOutlineFocus`       | `--input-outline-focus`        | `rgba(108, 95, 199, 1)`                   | `rgba(108, 95, 199, 1)`                   | Outline color for form inputs when focused                                                            |
+
+| `buttonBackground`        | `--button-background`          | `var(--background)`                       | `var(--background)`                       | Background color for the cancel button                                                                |
+| `buttonBackgroundHover`   | `--button-background-hover`    | `#f6f6f7`                                 | `#352f3b`                                 | Background color when hovering over the cancel button                                                 |
+| `buttonBorder`            | `--button-border`              | `var(--border)`                           | `var(--border)`                           | Border style for the cancel button                                                                    |
+| `buttonBorderRadius`      | `--button-border-radius`       | `var(--input-border-radius)`              | `var(--input-border-radius)`              | Border radius style for form buttons                                                                  |
+| `buttonOutlineFocus`      | `--button-outline-focus`       | `var(--input-outline-focus)`              | `var(--input-outline-focus)`              | Outline color for the cancel button, in the focused state                                             |
```

Comment:
and screenshot button too!

Reply:
changing to `The label of cancel buttons used in the feedback form.`